algorithms/kernel/dtrees/forest/classification/df_classification_predict_dense_default_batch_impl.i

```diff
@@ -62,6 +62,34 @@ typedef int32_t featureIndexType;
 #define _SCALE_FACTOR_FOR_VECT_PARALLEL_COMPUTE  0.3  /* scale tree size to chose whethever vectorized or not compute path in parallel mode */
 #define _MIN_NUMBER_OF_ROWS_FOR_VECT_SEQ_COMPUTE 1024 /* min number of rows to be predicted by vectorized compute path in sequential mode */
 
+template <typename algorithmFPType, CpuType cpu>
+DAAL_FORCEINLINE void fillResults(size_t nClasses, enum VotingMethod votingMethod, size_t blockSize,
+    const double * probas, const leftOrClassType * classes, const uint32_t * leafsIndexes,
+    algorithmFPType * resPtr) {
+    if (votingMethod == VotingMethod::unweighted || probas == nullptr)
+    {
+        PRAGMA_IVDEP
+        PRAGMA_VECTOR_ALWAYS
+        for (size_t i = 0; i < blockSize; i++)
+        {
+            const size_t cl = classes[leafsIndexes[i]];
+            resPtr[i * nClasses + cl]++;
```

Comment:
Here and in another lines of the file: use prefix increment (see coding guidelines).